spring-cloud-dataflow-server-local/src/main/resources/application.yml

```diff
@@ -10,7 +10,7 @@ spring:
       registry:
         populator:
           locations: classpath:META-INF/${binder}-apps.properties
-
+serverStore: redis
```

Comment:
Just curious why are we defaulting to redis as a serverStore if we have an embedded h2 that we can use.   Granted its still in-memory, but that means that a user could technically start a DF Server without a Redis instance running (unless they wanted to do analytics.)  Thoughts?